corehq/apps/data_vault/__init__.py

```diff
@@ -0,0 +1,22 @@
+from django.db import router
+
+from corehq.apps.data_vault.models import VaultStore
+
+VAULT_DB_NAME_FOR_WRITE = router.db_for_write(VaultStore)
```

Comment:
I don't think creating this as a module level var is a good idea since the routers may not be fully initialised by the time this is called

Reply:
> Ideally the model class should never be used outside of the vault package

this is what I was thinking about. Or is that restriciton only for method calls on the model?

Reply:
turns out i can't even load the app because of what i was doing in the __init__ file so had to do
https://github.com/dimagi/commcare-hq/pull/28161/commits/a2420f5b3319fb1a224f938e094e3330e40bc908